### DISCUSSION TITLE
I’d like to point out sysadmins have already discovered this, and are actively finding ways to block it.

### DISCUSSION THREAD

**Discussion by frogmasamune** [3kh0/ext-remover#273] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
https://www.reddit.com/r/k12sysadmin/comments/yetatv/anyone_else_dealing_with_ltbeef/?utm_source=share&utm_medium=web2x&context=3 

Are there any measures that can be implemented so you need to be at least logged in to see this? There’s no point to exploits if sysadmins can easily find and patch them.

**Comment by bartholemuew** [comment]
nerd admins 🤓

**Comment by ILikeBypassing** [comment]
use another device ez

**Reply by Aka-but-nice** [reply]
It's cringy when the admins said "ltbeef" like thats cringe tbh
